rust/src/mask/masking.rs

```diff
@@ -208,7 +211,7 @@ impl Aggregation {
 
         // If we have at least one object, make sure the object we're
         // trying to aggregate has the same length.
-        if self.nb_models > 0 && (self.object.data.len() != object.data.len()) {
+        if self.nb_models > 0 && (self.object_size != object.data.len()) {
```

Comment:
perhaps the `self.nb_models > 0` check can be removed? to prevent even the first object in the aggregate from being the wrong length...

Reply:
Good point👍 I will remove it